pom.xml

```diff
@@ -600,6 +600,12 @@
       <version>2.0.3</version>
       <scope>test</scope>
     </dependency>
+	<dependency>
```

Comment:
Used to facilitate loading a custom private JWK for a GHApp

Reply:
I considered relying on jjwt, but the feature does not exist yet into there, and people points back to nimbus-jose: https://github.com/jwtk/jjwt/issues/236#issuecomment-640715042

I would have to go through the JWK to PEM procedure, which is a burden given my own setup (i.e. my app using this lib relies on a JWK, not a PEM). Also, it would be more error-prone to rely on a given PEM content through environment variable (which is again my setup, and how I suggest to manage custom privateKey here).

I tried copying some classes from Nimbus to fill the gap, but it seems to require quite a bunch of classes. I'm having another try but focusing on the minimal set of methods to get a `PrivateKey`.

My only justification for this additional lib is existing tests for GitHub App APIs has not dynamic mechanisms for now (neither from local file, or from Env). I may then just drop this code, and be potentially discouraged to contribute here (this is a weak argument as I would be happy to fit into any existing mechanism, the current option would be to copy my own privateKey as a test resources, with a risk of pushing it publicly).

Reply:
I'm dropping nimbus, with a mechanism to load a PEM from a `Path` provided as environment variable (similarly to the pushed code). However, I will not be a user of this mechanism. It will pave the way for next contribution requiring a custom JWK.